<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QMainWindow" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>544</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>915</width>
    <height>544</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>915</width>
    <height>544</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/assets/images/editor.png);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="item_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>32</width>
      <height>128</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/assets/images/vertical_wood.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cleanButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 14pt &quot;Noto Rashi Hebrew ExtBd&quot;;
color: rgb(255, 255, 0);
background-image: transparent;
border-image: transparent;</string>
    </property>
    <property name="text">
     <string>Clean</string>
    </property>
   </widget>
   <widget class="QPushButton" name="item_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/assets/images/one_block.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 14pt &quot;Noto Rashi Hebrew ExtBd&quot;;
color: rgb(255, 255, 0);
background-image: transparent;
border-image: transparent;</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 14pt &quot;Noto Rashi Hebrew ExtBd&quot;;
color: rgb(255, 255, 0);
background-image: transparent;
border-image: transparent;</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="item_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/assets/images/horizontal_wood.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="item_1">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/assets/images/box.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="musicButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 13pt &quot;Noto Rashi Hebrew ExtBd&quot;;
color: rgb(255, 255, 0);
background-image: transparent;
border-image: transparent;</string>
    </property>
    <property name="text">
     <string>Music</string>
    </property>
   </widget>
   <widget class="QPushButton" name="item_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>32</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/assets/images/respawn.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="item_6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>32</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/assets/images/gunSpawner.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>cleanButton</zorder>
   <zorder>saveButton</zorder>
   <zorder>exitButton</zorder>
   <zorder>musicButton</zorder>
   <zorder>item_2</zorder>
   <zorder>item_1</zorder>
   <zorder>item_3</zorder>
   <zorder>item_4</zorder>
   <zorder>item_5</zorder>
   <zorder>item_6</zorder>
  </widget>
 </widget>
 <resources>
  <include location="editor_resource.qrc"/>
 </resources>
 <connections/>
</ui>
